src/controller/ReadInteraction.h

```diff
@@ -66,6 +74,11 @@ CHIP_ERROR ReportAttribute(Messaging::ExchangeManager * exchangeMgr, EndpointId
     if (readClient->IsSubscriptionType())
     {
         readPaths.release();
+        if (aDataVersion.HasValue())
+        {
+            dataVersionFilters.release();
```

Comment:
Could be unconditional, right?  Releasing a null unique_ptr is fine.

Reply:
agree, thanks